src/Ng/App.config

```diff
@@ -10,7 +10,7 @@
     <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
       <dependentAssembly>
         <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30ad4fe6b2a6aeed" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-8.0.0.0" newVersion="8.0.0.0" />
+        <bindingRedirect oldVersion="0.0.0.0-9.0.0.0" newVersion="9.0.0.0" />
```

Comment:
This isn't strictly related to this PR, but do we want to move this package (newtonsoft.json) to the 9.x release for all of our packages as we update them? I think it would be best to be consistent one way or the other, and I would prefer to lean towards updating if it's not too much of as hassle (which you've already done here).